lib/ansible/playbook/task.py

```diff
@@ -78,7 +78,7 @@ class Task(Base, Conditional, Taggable, CollectionSearch):
     _delegate_facts = FieldAttribute(isa='bool')
     _failed_when = FieldAttribute(isa='list', default=list)
     _loop = FieldAttribute()
-    _loop_control = FieldAttribute(isa='class', class_type=LoopControl, inherit=False)
+    _loop_control = FieldAttribute(isa='class', class_type=LoopControl, inherit=False, default=LoopControl)
```

Comment:
should you have to explicitly default to the class? I think this might be a bug in the FA code